R/03-geojob-obj.R

```diff
@@ -56,6 +56,31 @@ setMethod("geojob", signature(), function(...) {
   return(geojob)
 })
 
+#'@param xml location of xml 
+#'@rdname geojob-methods
+#'@importFrom httr GET 
+#'@importFrom RCurl url.exists
+#'@importFrom XML xmlTreeParse xmlAttrs xmlRoot
+#'@aliases geojob,geojob-method
+setMethod("geojob", signature("ANY"), function(xml, urlSlot = NULL) {
+  ## create new geojob object
+  job <- new("geojob")
+  #parse based on xml class
+  if(is.character(xml)) {
+    if(url.exists(xml)){
+      doc <- GET(xmlLocation)
+    } 
+    doc <- xmlTreeParse(doc)
+  } 
+  #fill slots
+  xml(job) <- toString.XMLNode(doc$doc$children[[1]])
```

Comment:
instead of doing `job <- new()` above, I think you should create it here w/ 
```r
job <- geojob(xml = thing, version = other.thing, url = urlSlot)
```